src/Applications/LDAS_App/ldas_setup

```diff
@@ -194,6 +196,18 @@ class LDASsetup:
                                                _datediff.minutes,
                                                _datediff.seconds)
             self.job_sgmt.append("JOB_SGMT: "+self.rqdExeInp['JOB_SGMT'])
+
+        if 'NUM_SGMT' not in self.rqdExeInp:
+           self.rqdExeInp['NUM_SGMT'] = 1
+        # hard set NUM_SGMT and NUM_SGMT
+        if (self.ladas_coupling > 0) :
+           if self.rqdExeInp['NUM_SGMT'] != 1 :
+              print "WARNING: 'NUM_SGMT' is reset to 1"
+              self.rqdExeInp['NUM_SGMT'] = 1 
+           if self.rqdExeInp['JOB_SGMT'] != "00000000 060000" :
+              print "WARNING: 'JOB_SGMT' is reset to  00000000 060000"
+              self.rqdExeInp['JOB_SGMT'] = "00000000 060000" 
```

Comment:
ldas_setup should exit here, rather than overwrite the user's setting (see previous comment)